### DISCUSSION TITLE
有什么办法可以让table 的cell的 tooltip 就在当前位置显示吗?

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#42430] [Q&A] [answered]
https://rsuitejs.com/components/table/#show-full-text-of-cells

想实现这样的效果

**Comment by BoyYangzai** [comment, marked as answer]
@vaynevayne  试试通过 `mouseEnter` 拿到当前元素然后改变该 `cell` 的样式
https://codesandbox.io/s/zi-ding-yi-dan-yuan-ge-sheng-lue-ti-shi-antd-5-5-0-forked-tsvixl?file=/demo.tsx
![image](https://github.com/ant-design/ant-design/assets/94534613/cef7d214-a1e1-4f2d-9e25-f153182a5907)
![image](https://github.com/ant-design/ant-design/assets/94534613/5b4156bd-ce2c-4efb-bbb1-f81d81a11a35)

**Reply by vaynevayne** [reply]
666
